src/common/components/molecules/ViewSelector.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Button } from "@/common/components/atoms/button";
+import { FaList, FaThLarge } from "react-icons/fa";
+
+export type ViewMode = "list" | "grid";
```

Comment:
It might be nice to let this be free form and define the "ViewMode"s as part of the API, but that's a nice future idea. Not a now thing